packages/models/proto/v2/authorization/client.proto

```diff
@@ -0,0 +1,23 @@
+syntax = "proto3";
+
+package authorization.v2;
+
+import "v2/authorization/key.proto";
+
+message ClientV2 {
+  string id = 1;
+  string consumerId = 2;
+  string name = 3;
+  optional string description = 4;
+  repeated string relationships = 5;
```

Comment:
no more required (remember to update positions accordingly)
```suggestion
```

Reply:
Removed